src/test/java/com/tjtanjin/steve/tasks/TaskTest.java

```diff
@@ -0,0 +1,47 @@
+package com.tjtanjin.steve.tasks;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import java.time.LocalDate;
+
+import org.junit.jupiter.api.Test;
+
+public class TaskTest {
+    private final Task task = new Task("Test Task", "incomplete", "TODO");
```

Comment:
TASK?

Reply:
Thank you for pointing this out, fixed in latest commit! :)